app/assets/javascripts/components/admin_panel/admin_header.es6.jsx

```diff
@@ -0,0 +1,33 @@
+class AdminHeader extends React.Component {
+
+  render () {
+    return (
+    <div>
+      <Navbar
+        fixedTop={true}
+        className="header">
+        <Navbar.Header>
+            <Navbar.Brand>
+              <img className="header__logo" src={ImageConstants.logos.orange} href="#" />
```

Comment:
![image](https://cloud.githubusercontent.com/assets/8731793/19373210/39e6c0d8-9177-11e6-959c-0bac30e2296a.png)

Can you copy this so that when you click the forte logo, it goes back to the home page?